SpatialGDK/Source/SpatialGDKTests/SpatialGDK/EngineClasses/SpatialVirtualWorkerTranslationManager/SpatialOSDispatcherMock.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/SpatialOSDispatcherInterface.h"
+
+// The SpatialOSDispatcherMock is intended as a very minimal implementation which will acknowledge
+// and record any calls. It can then be used to unit test other classes and validate that given
+// particular inputs, they make calls to SpatialOS which are as expected.
+//
+// Currently, only a few methods below have implementations. Feel free to extend this mock as needed
+// for testing purposes.
+
+class SpatialOSDispatcherMock : public SpatialOSDispatcherInterface
```

Comment:
`SpatialOSDispatcherMock` looks more like a Spy rather than Mock to me (according to `SpatialGDKTestGuidelines.h`).

Reply:
I agree. I hadn't actually seen the Spy terminology before, but it definitely matches, I'll rename.